includes/REST_API/Embed_Controller.php

```diff
@@ -198,8 +222,12 @@ private function get_data_from_post( $url ) {
 
 			// In case of subdirectory configs, set the path.
 			if ( ! is_subdomain_install() ) {
+				// Get "sub-site" part of "http://example.org/sub-site/web-stories/my-story/".
+				// But given just "http://example.org/web-stories/my-story/", don't treat "web-stories" as site path.
+				// This differs from the logic in get_oembed_response_data_for_url() which does not do this.
+				// TODO: Investigate possible core bug in get_oembed_response_data_for_url()?
 				$path    = explode( '/', ltrim( $url_parts['path'], '/' ) );
-				$path    = reset( $path );
+				$path    = count( $path ) > 2 ? reset( $path ) : false;
```

Comment:
```suggestion
				$path    = count( $path ) > 2 ? reset( $path ) : '';
```
Why not default to an empty string?

Reply:
Doesn't matter too much, just needs to be falsey for the check below.

Reply:
Empty string is a default value of get_networks.

Reply:
Right, but `$path is only used below when `if ( $path && $network instanceof \WP_Network )` evaluates to true. So this won't make a difference.

I can change it though, no problem. Prevents accidental bugs in the future 👍  Just wanted to point that out :)